sdk/dnsresolver/Azure.ResourceManager.DnsResolver/CHANGELOG.md

```diff
@@ -1,22 +1,28 @@
 # Release History
 
-## 1.2.0-beta.1 (2024-10-18)
+## 1.2.0-beta.1 (2024-10-24)
 
 ### Features Added
 
-- Enable the new model serialization by using the System.ClientModel, refer this [document](https://aka.ms/azsdk/net/mrw) for more details.
+- Upgraded api-version tag from 'package-2022-07' to 'package-preview-2023-07'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/b26a190235f162b15d77dad889d104d06871fb4f/specification/dnsresolver/resource-manager/readme.md.
```

Comment:
do we need to specify the tag detail in the changelog later on?

Reply:
No, we have discussed before that don't want to have details of api change in changelog.